### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by dmsolr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,80 @@
...
+
+[![Twitter Follow](https://img.shields.io/twitter/follow/asfskywalking.svg?style=for-the-badge&label=Follow&logo=twitter)](https://twitter.com/AsfSkyWalking)
+
+![CI](https://github.com/apache/skywalking-nginx-lua/workflows/CI/badge.svg?branch=master)
+
+
+[**SkyWalking**](https://github.com/apache/skywalking) Agent Test Tool is a tremendously useful test tools suite in a wide variety of languages of `Agent`. 
+Includes `mock collector` and `validator`. The `mock collector` is a SkyWalking receiver, likes OAP server. 
+
+The `mock collector` is responsible for mocking the SkyWalking OAP receiver to collect data from the agent. It will collect data from Agent reported by `GRPC`.
```
Comment (wu-sheng) [inline README.md]: Remove `will`, Does this support HTTP? `it receives data from agent through GRPC and HTTP`?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,80 @@
...
+
+![CI](https://github.com/apache/skywalking-nginx-lua/workflows/CI/badge.svg?branch=master)
+
+
+[**SkyWalking**](https://github.com/apache/skywalking) Agent Test Tool is a tremendously useful test tools suite in a wide variety of languages of `Agent`. 
+Includes `mock collector` and `validator`. The `mock collector` is a SkyWalking receiver, likes OAP server. 
+
+The `mock collector` is responsible for mocking the SkyWalking OAP receiver to collect data from the agent. It will collect data from Agent reported by `GRPC`. 
+And then it can be downloaded by Http API that the mock collector collected(service registered, instance registered, and segments). 
+Finally, to validate the file downloaded from `mock collector` by `SkyWalking Validator.
```
Comment (wu-sheng) [inline README.md]:
```suggestion
Eventually, validate the file downloaded from `mock collector` by using `SkyWalking Validator`. 
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,80 @@
...
+```
+
+## Apache SkyWalking Validator Tool
+
+`Validator Tool` a data validating tool. It resposible for validating the `expected data` with `actual data`.
+
+```bash
+java -jar \
+    -Xmx256m -Xms256m \
+    -DcaseName="${SCENARIO_NAME}-${SCENARIO_VERSION}" \
```
Comment (wu-sheng) [inline README.md]: This parameter value demonstration is not clear, should be `path/to/expected-file-folder`?

File: README.md [hunk 4]
```diff
@@ -1,7 +1,80 @@
...
+    -DtestCasePath=${SCENARIO_HOME}/data/ \
```
Comment (wu-sheng) [inline README.md]: `path/to/download-folder` ?

File: README.md [hunk 5]
```diff
@@ -1,7 +1,80 @@
...
+
+after above steps, we can check whether the `mock collector` is available throgh HTTP API. To visit `http://localhost:12800/status` and get the reponse with `success` in the body of content.
+
+Finally, To visit follow URL to download the data as `yaml` file, which can be used by `SkyWalking Validator Tool`.
+
+http://localhost:12800/receiveData
+
+- How to config on agent
+
+We can modify directly the conguration of agent, `./config/agent.conf`, likes following.
```
Comment (kezhenxu94) [inline README.md]:
```suggestion
We can modify the configurations of agent in `./config/agent.conf`, as follows
```

File: README.md [hunk 6]
```diff
@@ -1,7 +1,80 @@
...
+
+```yaml
```
Comment (kezhenxu94) [inline README.md]: I don't think it's yaml

File: README.md [hunk 7]
```diff
@@ -1,7 +1,80 @@
...
+or specify the jvm arguments likes:
+
+```bash
+# Backend service addresses.
+-DSW_AGENT_COLLECTOR_BACKEND_SERVICES=127.0.0.1:19876
+```
+
+## Apache SkyWalking Validator Tool
+
+`Validator Tool` a data validating tool. It resposible for validating the `expected data` with `actual data`.
```
Comment (kezhenxu94) [inline README.md]:
```suggestion
`Validator Tool` is a data validating tool. It is responsible for validating the `expected data` with `actual data`.
```

**Pull request merged**